kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/logs/TestLogCollector.java

```diff
@@ -0,0 +1,232 @@
+ *     │              │          │          └── my-cluster.yaml
+ *     │              │          ├── kafkanodepool
+ *     │              │          │          └── kafka.yaml
+ *     │              │          ├── events.log
+ *     │              │          ├── pod
+ *     │              │          │          ├── describe-pod-my-cluster-kafka-0.log
+ *     │              │          │          ├── describe-pod-my-cluster-kafka-1.log
+ *     │              │          │          ├── describe-pod-my-cluster-kafka-2.log
+ *     │              │          │          └── logs-pod-strimzi-cluster-operator-57d455495c-98t8f-container-strimzi-cluster-operator.log
+ *     │              │          └── secret
+ *     │              │              ├── builder-dockercfg-j99st.yaml
+ *     │              │              ├── default-dockercfg-pb9z6.yaml
+ *     │              │              ├── deployer-dockercfg-ltvsj.yaml
+ *     │              │              └── strimzi-cluster-operator-dockercfg-64p64.yaml
+ *     │              └── kafka-XXXXXX
+ *     │                  ├── configmap
+ *     │                  │          ├── cluster-2acba704-b-6eac3ce0-0.yaml
+ *     │                  │          ├── cluster-2acba704-bridge-config.yaml
+ *     │                  │          ├── cluster-2acba704-c-6eac3ce0-1.yaml
+ *     │                  │          ├── cluster-2acba704-entity-topic-operator-config.yaml
+ *     │                  │          ├── cluster-2acba704-entity-user-operator-config.yaml
+ *     │                  │          ├── regcred.yaml
+ *     │                  │          └── openshift-service-ca.crt.yaml
+ *     │                  ├── deployment
+ *     │                  │          ├── cluster-2acba704-bridge.yaml
+ *     │                  │          └── cluster-2acba704-entity-operator.yaml
+ *     │                  ├── events.log
+ *     │                  ├── job
+ *     │                  │          ├── admin-client-cli-create.yaml
+ *     │                  │          ├── admin-client-cli-delete.yaml
+ *     │                  │          ├── ...
+ *     │                  │          └── kafka-producer-client-zzzzzzz.yaml
+ *     │                  ├── pod
+ *     │                  │          ├── describe-pod-kroxylicious-proxy-7db664df48-bxxrx.log
+ *     │                  │          ├── ...
+ *     │                  │          └── logs-pod-kroxylicious-proxy-7db664df48-bxxrx-container-kroxylicious.log
+ *     │                  └── secret
+ *     │                      └── regcred.yaml
+ * ...
+ */
+public class TestLogCollector {
+    private static final String CURRENT_DATE;
+    private final LogCollector logCollector;
+
+    static {
+        // Get current date to create a unique folder
+        DateTimeFormatter dateTimeFormatter = DateTimeFormatter.ofPattern("yyyy-MM-dd-HH-mm-ss");
+        dateTimeFormatter = dateTimeFormatter.withZone(ZoneId.of("GMT"));
+        CURRENT_DATE = dateTimeFormatter.format(LocalDateTime.now());
```

Comment:
This looks weird, firstly the name says DATE when it is actually a date time.  Second, it is the current date only when the moment the JVM loads the class.   It is stale after that point.